core/src/test/java/org/apache/calcite/avatica/util/TimeAccessorTest.java

```diff
@@ -46,77 +49,74 @@ public class TimeAccessorTest {
    */
   @Before public void before() {
     final AbstractCursor.Getter getter = new LocalGetter();
-    localCalendar = Calendar.getInstance(TimeZone.getDefault(), Locale.ROOT);
-    instance = new AbstractCursor.TimeAccessor(getter, localCalendar);
+    localCalendar = Calendar.getInstance(IST_ZONE, Locale.ROOT);
+    instance = new AbstractCursor.TimeAccessor(getter, localCalendar, false);
   }
 
   /**
-   * Test {@code getTime()} returns the same value as the input time for the local calendar.
+   * Test {@code getTime()} returns the same value as the input time for the connection default
+   * calendar.
```

Comment:
Is there a bug in Avatica? If so, say what it is. If not, why are you changing/removing tests?

Reply:
This was just me trying to reconcile the test description with the logic therein, though I realize now that I didn't do a great job of it. The test says "for the local calendar", but it's unclear if that means `localCalendar` (which is not involved in this test at all), or UTC (which in most cases would not be accurately described as "the local calendar"). Upon closer inspection I can appreciate that this test case is really meant to test that no adjustment occurs when the explicitly provided calendar is either UTC or null. I'm working to clean up these tests in general, and in this case I think the I should've just updated the javadoc.